.github/workflows/latency-tracking.yml

```diff
@@ -31,9 +28,9 @@ jobs:
           cd latency
           docker build --build-arg JINA_VER=master . -t latency-tracking
           docker run -v $(pwd)/output:/workspace/output -v $(pwd)/original:/workspace/original latency-tracking
-          bash batch.sh 1
+          bash batch.sh ${{ env.NUM_LAST_RELEASE }}
           pip install prettytable
-          python ppstat.py > comment.txt
+          python ppstat.py ${{ env.NUM_LAST_RELEASE }}>  comment.txt
```

Comment:
what does this `ppstat` do?

Reply:
part of `latency-tracking` repo, for pretty print stats so that we have a nice table view on GIthub thread